homeassistant/components/rest/sensor.py

```diff
@@ -197,13 +197,16 @@ def update(self):
             if value:
                 try:
                     json_dict = json.loads(value)
+                    if isinstance(json_dict, list):
+                        json_dict = json_dict[0]
                     if isinstance(json_dict, dict):
                         attrs = {
                             k: json_dict[k] for k in self._json_attrs if k in json_dict
                         }
                         self._attributes = attrs
                     else:
-                        _LOGGER.warning("JSON result was not a dictionary")
+                        _LOGGER.warning("JSON result was not a dictionary" +
```

Comment:
`+` is not needed.

Reply:
Removed (a coupled days back, but want to document that it's fixed)